cardano-chain-gen/test/Test/Cardano/Db/Mock/Unit/Babbage/Config/MigrateConsumedPruneTxOut.hs

```diff
@@ -385,7 +341,6 @@ bootstrapRestartMissingFlag = do
   where
     cmdLineArgs =
       initCommandLineArgs
-        { claConfigFilename = "test-db-sync-config-bootstrap.json"
-        , claBootstrap = True
+        { claBootstrap = True
```

Comment:
With bootstrap moved to config files, I think this is currectly never used. It doesn't need to be done in this pr, since it's a separate issue.